doc/source/whatsnew/v1.4.0.rst

```diff
@@ -757,6 +757,8 @@ Timedelta
 ^^^^^^^^^
 - Bug in division of all-``NaT`` :class:`TimeDeltaIndex`, :class:`Series` or :class:`DataFrame` column with object-dtype arraylike of numbers failing to infer the result as timedelta64-dtype (:issue:`39750`)
 - Bug in floor division of ``timedelta64[ns]`` data with a scalar returning garbage values (:issue:`44466`)
+- Bug in :class:`Timedelta` now properly taking into account any nanoseconds contribution of any kwarg (:issue:`43764`)
```

Comment:
This is form somewhere else?

Reply:
It looks like something was merged between my writing up the PR and submitting it. I just did a fetch and merge on my branch.